cybersyn/scripts/lib.lua

```diff
@@ -71,3 +73,11 @@ function dual_pairs(t1, t2)
 		return key
 	end
 end
+
+--- @param count number
+--- @return string
+function format_signal_count(count)
+	local formatted = util.format_number(count, true)
```

Comment:
`util.format_number` formats differently than `LuaGuiElement.number` (`B` instead of `G`, `10.3k` instead of `10k`)
I propose this instead:

```lua
--- @param count integer
--- @return string
function format_signal_count(count)
	local function si_format(divisor, si_symbol)
		count = math.floor(count / divisor)
		local format = (math.abs(count) >= 10) and "%.0f%s" or "%.1f%s"
		return string.format(format, count, si_symbol)
	end

	local abs = math.abs(count)
	if abs >= 1000000000 then -- signals are 32bit integers so Giga is enough
		return si_format(1000000000, "G")
	end
	if abs >= 1000000 then
		return si_format(1000000, "M")
	end
	if abs >= 1000 then
		return si_format(1000, "k")
	end
	return tostring(count)
end
```